backend/app/api/v1/endpoints/cpt/cptJobs.py

```diff
@@ -0,0 +1,89 @@
+import json
+from fastapi import Response
+import pandas as pd
+from datetime import datetime, timedelta, date
+from fastapi import APIRouter
+from .maps.ocp import ocpMapper
+from ...commons import responses
+from fastapi.param_functions import Query
+
+
+router = APIRouter()
+
+products = {
+            "ocp": ocpMapper,
+           }
+
+
+response_example ={
+  "startDate": "2023-11-18",
+  "endDate": "2023-11-23",
+  "results": [
+        {
+        "ciSystem": "PROW",
+        "uuid": "f6d084d5-b154-4108-b4f7-165094ccc838",
+        "releaseStream": "Nightly",
+        "jobStatus": "success",
+        "buildUrl": "https://ci..org/view/1726571333392797696",
```

Comment:
It is better to have all the response metadata should be at one place. Can 
 we shift this metadata to responses.py